nautobot/core/templates/generic/object_list.html

```diff
@@ -3,35 +3,19 @@
 {% load helpers %}
 {% load static %}
 
-
-{% block extra_styles %}
-<style>
-    .btn-filter-value {
-        background: #cfcfcf;
-        margin-right: 5px;
-    }
-    .filters-applied>.btn {
-        margin-right: 5px;
-        margin-bottom: 5px;
-    }
-</style>
-{% endblock extra_styles %}
-
 {% block header %}
     <div class="row noprint">
-        {% with list_url=content_type.model_class|validated_viewname:"list" %}
         <div class="col-sm-8 col-md-9">
```

Comment:
Nit, but in the case where we don't have a search_form, there's no point in leaving a blank space for it - this should be simply `col` in that case so it spans the full width.